jacscript/src/busenv.ts

```diff
@@ -23,17 +24,20 @@ export interface JacsEnvOptions {
     setting?: (name: string) => Setting
 }
 
-export class JDBusJacsEnv implements JacsEnv {
+export class JDBusJacsEnv extends JDClient implements JacsEnv {
```

Comment:
maybe we should rename JDClient to JDMountable or something?

Reply:
Yes JDComoponrnt